processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -416,7 +421,16 @@ public InputRow formatRow(InputRow row)
    */
   public int add(InputRow row) throws IndexSizeExceededException {
     TimeAndDims key = toTimeAndDims(row);
-    final int rv = addToFacts(metrics, deserializeComplexMetrics, row, numEntries, key, in, rowSupplier);
+    final int rv = addToFacts(
+        metrics,
+        deserializeComplexMetrics,
+        reportParseExceptions,
```

Comment:
minor nit: It makes it a bit easy to backport and fix conflicts if the newly added parameter is the last one in method.